app/lib/rendering/OpenTypePen.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2015, Felipe Correa da Silva Sanches <juca@members.fsf.org>
+ * 
+ * You should have received a copy of the MIT License along with this program.
+ * If not, see http://www.opensource.org/licenses/mit-license.php
```

Comment:
Metapolator is GPL, and maybe we should make ufoJS GPL, too. We'll have to have a discussion about this.